pages/common/pueue-send.md

```diff
@@ -0,0 +1,16 @@
+# pueue send
+
+> Send input to a task.
+> More information: <https://github.com/Nukesor/pueue>.
+
+- Send input to a running command:
+
+`pueue send {{task_id}} "{{input}}"`
+
+- Send confirmation to a task expecting y/N (e.g. apt, cp):
+
+`pueue send {{task_id}} {{y}}`
+
+- Write to a file open in background:
+
+`pueue add cat > {{location/to/file}}; sleep 3h; pueue send {{task ID}} {{some text}}`
```

Comment:
@she3o I don't think this example is necessary either.  The `pueue send` command is very simple and I think one example is probably fine for the page.

Reply:
I'm also a little confused as to what this command does&mdash;if you could explain it I'd be fine with keeping (if it makes sense to have and isn't a duplicate ;)